WordPress/gutenberg
# Support `colspan` attribute in `table` HTML, including when pasting

## What?
Support `colspan` attribute on `th` and `td` when pasting tables in the Editor. 

## Why?
It was based on [this](https://github.com/WordPress/gutenberg/issues/45774#issuecomment-1316052739) requirement in https://github.com/WordPress/gutenberg/issues/45774

## How?


## Testing Instructions





1. Check out this branch and try pasting tables in from external sources. The `colspan` attribute should be supported on `th` and `td` elements within `thead`, `tbody` and `tfoot` elements. 

## Screenshots or screencast 

![image](https://user-images.githubusercontent.com/6851384/203436468-71d58390-a9b1-4e86-9427-bb88cfa7241a.png)
